test/OSSObject.test.ts

```
@@ -22,7 +22,8 @@
   const __filename = fileURLToPath(import.meta.url);
   const __dirname = path.dirname(__filename);
 
-  describe('list()', () => {
+  // OSSClientError: Access denied by bucket policy.
+  describe.only('list()', () => {
```

Biome

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)